docs/deployment.md

```diff
@@ -14,7 +14,21 @@ To deploy to production server:
   git submodule init && git submodule update
   ```
   Repository details see [Project repository](git-repository.md).
-3. Link production configuration for docker-compose, Details see [Docker container bootstrapping](../README.md#initializationdocker-container-bootstrapping).
+3. a. Link production configuration for docker-compose, Details see [Docker container bootstrapping](../README.md#initializationdocker-container-bootstrapping).
+   b. Edit the docker-compose.yml file and change (if needed) the EMAIL settings:
+   ```shell
+    DJANGO_EMAIL_HOST='<your smtp>'
+    DJANGO_EMAIL_HOST_PASSWORD='<your password>'
+    DJANGO_EMAIL_HOST_USER='<your user>'
+    DJANGO_EMAIL_PORT=<PORT>
+    DJANGO_EMAIL_SUBJECT_PREFIX='<set to something smart>'
+    DJANGO_EMAIL_USE_TLS=<true/false>
```

Comment:
You mean the true/false? [No](https://github.com/joke2k/django-environ/blob/develop/environ/environ.py#L284).